content/java-interview-questions-2/4-new-java-features.md

```diff
@@ -1,37 +1,96 @@
 ## Question
-# What is Spring?
+# Which version of Java have you worked on ?
+--
+# Answer
+- you can explain whatever java version you have used like java 8,9,10 or 11.
+- out of these java 8 and java 11 are the long term support versions and most of the questions will be on these 2 versions.
+---
```

Comment:
I'd skip this question. The LTS point is interesting, so maybe this can be a separate question